includes/ApplHandlingByStr.php

```diff
@@ -33,6 +34,46 @@ function showStatus( $stat_str )
 		{
 			echo "<td><b>REJECTED</b> by ".$str[0] ."</td>";
 		}
+		else if( $str[1] == 'F' )
+		{
+			echo "<td><b>FORWARDED</b> from ".$str[0]."</td>";
+		}
+		else
+		{
+			echo "<td><b>Unknown Status</b></td>";
+		}
+	}
+}
+
+
+function showFullStatus( $stat_str )
```

Comment:
__[code_sniffer]__
* Expected 0 spaces between opening bracket and argument "$stat_str"; 1 found
* Expected 0 spaces between argument "$stat_str" and closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->